docs/book/src/developer/providers/v1.2-to-v1.3.md

```diff
@@ -18,7 +18,7 @@ in Cluster API are kept in sync with the versions used by `sigs.k8s.io/controlle
 
 ### Deprecation
 
--
+- `controllers.external.CloneTemplate` has been deprecated and will be removed in a future release. Please use `controllers.external.CreateFromTemplate` instead.
```

Comment:
```suggestion
- `controllers/external.CloneTemplate` has been deprecated and will be removed in a future release. Please use `controllers/external.CreateFromTemplate` instead.
```

Reply:
Don't know if this would be more correct, or even `sigs.k8s.io/cluster-api/controllers/external.CloneTemplate` etc.